modules/openapi-generator/src/main/resources/php/ObjectSerializer.mustache

```diff
@@ -307,6 +314,7 @@ class ObjectSerializer
             }
         }
 
+        /** @psalm-suppress ParadoxicalCondition */
```

Comment:
This one is interesting, Psalm found it's nonsensical since it includes `object`, which is already handled a few lines above, meaning this branch can never be reached with that specific "class".

This introduces the first tool-specific annotation, but I believe it's not a big issue since Doctrine and PHPUnit already use them natively.

Reply:
See example what it found here: https://psalm.dev/r/1035490c06